Formula/mvtools.rb

```diff
@@ -3,19 +3,10 @@ class Mvtools < Formula
   homepage "https://github.com/dubhater/vapoursynth-mvtools"
   url "https://github.com/dubhater/vapoursynth-mvtools/archive/v23.tar.gz"
   sha256 "3b5fdad2b52a2525764510a04af01eab3bc5e8fe6a02aba44b78955887a47d44"
-  license "GPL-2.0"
-  revision 1
+  license "GPL-2.0-only"
+  revision 2
   head "https://github.com/dubhater/vapoursynth-mvtools.git", branch: "master"
 
-  bottle do
```

Comment:
Please don't touch the `bottle` block.

Reply:
Sorry, I read in the docs somewhere the bottle part would be added automatically and you shouldn't include it. Maybe I misread and that applied only to new formula

Reply:
You shouldn't touch it at all. If it's a new formula, don't add it. If it's an existing formula, don't  modify it.